components/Storyblok/SbTypeform.tsx

```diff
@@ -0,0 +1,162 @@
+import { storyblokEditable } from '@storyblok/react/rsc';
+import {
+ PopOver, PopUp, SideTab, Slider, Widget,
+} from '@/components/TypeForm';
+import { Heading } from '@/components/Typography';
+import * as CTAStyles from '@/components/Cta/Cta.styles';
+import { cnb } from 'cnbuilder';
+
+export type SbTypeformProps = {
+  blok: {
+    _uid: string;
+    id: string;
+    autoFocus: false;
+    component: string;
+    embedType: 'slider' | 'popup' | 'popover' | 'sidetab' | 'widget';
+    autoResize: boolean;
+    fullScreen: boolean;
+    heading: string;
+    hideFooter: boolean;
+    hideHeader: boolean;
+    buttonLabel: string;
+    embedHeight: string;
+    disableScroll: boolean;
+    enableSandbox: boolean;
+    transitiveSearchParams: string;
+    displayAsFullScreenModal: boolean;
+  };
+};
+
+// TODO: Move this to the non-storyblok Typeform component dir.
+export const styles = {
+  container: 'cc flex flex-col place-items-center rs-py-6',
+  heading: 'px-[40px] tf-sm:px-[80px] max-w-[878px]',
+  fauxCTA: cnb(CTAStyles.cta, CTAStyles.ctaVariants.solid, CTAStyles.ctaSizes.large),
+};
```

Comment:
Some basic styles. We can move this somewhere else if/when things change.

Reply:
I will likely take away some of the layout styles here later on, since I'm going to expect this form to be nested inside a Section, then a Grid (which provides option for bounding width - cc or edge to edge, and # of columns within the bounding box). Happy to leave these in for now - I'll get to the stylings later 😄